tensorflow/lite/kernels/internal/tensor.h

```diff
@@ -82,6 +94,7 @@ class VectorOfTensors {
 
```

Comment:
Only updating data pointer but not updating shapes may be dangerous. The shape can be changed between invocations.